sources/packages/backend/apps/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -110,3 +110,8 @@ export enum SuccessWaitingStatus {
   Success = "Success",
   Waiting = "Waiting",
 }
+
+export class ApplicationNumberParamAPIInDTO {
+  @MaxLength(10)
```

Comment:
Since this one it is not a search parameter and we know the format, we can try to validate it further, for instance, it must be 10 characters long. Anything from 1 to 9 we can return a bad request.